app/components/board-btn/board-btn.js

```diff
@@ -0,0 +1,11 @@
+import { div, empty, button } from '_core/virtual-dom'
+import { link } from '_components/router/router'
+
+export default ({ name, url }) => (
+  link({
+    className: 'board-btn',
+    },{
+    route: url,
+    content: name
+  })
+)
```

Comment:
if the div, empty and button import are not used, please remove

Reply:
report feeder